docs/course_materials/exercises/03_Variables_and_datatypes/test_homework.py

```diff
@@ -0,0 +1,136 @@
+""" Automatically check student's homework
+
+The tests will check:
+ 1. The content of the exercise file
+ 2. The execution and output of the exercise file
+ 3. The output with different inputs
+
+Guidelines:
+- Student's exercise files should be placed in the same dir as this test.
+- Test names are in the form: test_<topic>_<exc>.py
```

Comment:
it's not Ideal, 
I would change it so I could give a file path to this test

Reply:
Sure. Will add that.